modules/nf-core/ilastik/pixelclassification/main.nf

```diff
@@ -0,0 +1,50 @@
+process ILASTIK_PIXELCLASSIFICATION {
+    tag "$meta.id"
+    label 'process_single'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "ILASTIK_MULTICUT module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+    container "labsyspharm/mcmicro-ilastik:1.6.1"
+
+    input:
+    tuple val(meta), path(h5)
+    tuple val(meta2), path(ilp)
+
+    output:
+    tuple val(meta), path("*.h5") , emit: h5
```

Comment:
```suggestion
    tuple val(meta), path("*.${suffix}") , emit: output
```

And here we define the extension of the output file. It would also be better to change the output name to something general (like `output` here in my suggestion since it can be any of the possible formats now)

Reply:
Can you also fix the name in the `meta.yml` if you've chosen how to name your output file?